src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -0,0 +1,130 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('material.components.fabSpeedDial', [])
+    .directive('mdFabSpeedDial', MdFabSpeedDialDirective);
```

Comment:
I was basing this off of the existing directives I saw such as `MdButtonDirective` and `MdBottomSheetDirective`.